### PR TITLE
Project info text and URL made configurable

### PR DESCRIPTION
**Pull request by shreyasshivakumara**
This pull request enables project text and URL configurable on the sda-auth frontend page.

### CODE REVIEW

**Review by pontus — DISMISSED** [review]

**Comment by pontus** [comment]
Looks fine to me.

**Review by pontus — APPROVED** [review]

File: charts/sda-svc/values.yaml [hunk 1]
```diff
@@ -156,6 +156,10 @@ global:
     corsMethods: ""
     # @param corsCreds, allow credentials in the request, cors is disabled if false
     corsCreds: false
+    # @param infoText, name of the project
+    infoText: "Example project"
+    # @param inforURL, URL of the project
+    inforURL: " https://example.com/project"
```
Comment (jbygdell) [inline charts/sda-svc/values.yaml:162]:
```suggestion
    # @param infoText, name of the project
    infoText: "About Federated EGA"
    # @param inforURL, URL of the project
    inforURL: "https://ega-archive.org/about/projects-and-funders/federated-ega/"
```

Either the fields should have proper defaults *or* be empty in the values file and validation done in `auth-deploy.yaml` to ensure that they are not empty.

**Review by jbygdell — CHANGES_REQUESTED** [review]
Just a

File: charts/sda-svc/values.yaml (hunk 1 above)
Comment (shreyasshivakumara) [reply charts/sda-svc/values.yaml:162]: Sure, I will add the default values as you suggested.

**Review by pontus — APPROVED** [review]

**Review by jbygdell — APPROVED** [review]

**Comment by aaperis** [comment]
@shreyasshivakumara @pontus  @jbygdell, this doesnot seem to work for me at least locally. (This has not been yet deployed on any cluster as far as I see)

**Comment by jbygdell** [comment]
> @shreyasshivakumara @pontus @jbygdell, this doesnot seem to work for me at least locally. (This has not been yet deployed on any cluster as far as I see)

Because the container image created from this PR has the tag: v0.0.96 and that is not deployed yet anywhere.

**Comment by aaperis** [comment]
> Because the container image created from this PR has the tag: v0.0.96 and that is not deployed yet anywhere.

Yes, which I guess we should refrain from until we're sure this works as expected.

**Comment by pontus** [comment]
Thanks, I'll try to check it out. @aaperis can you give me a hint so I'll know what to look for?

**Pull request merged**